### ISSUE TITLE
Item name in the ArrayField heading does renders undefined

### ISSUE THREAD

**Issue by obliviga** [department-of-veterans-affairs/va-forms-system-core#447]
Steps to reproduce:

1. http://localhost:3001/health-care/apply/application/review-and-submit
2. Insurance information
3. Edit Other Coverage
4. Do you have health insurance -> yes
5. Shows undefined: ![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/2bac0fd9-e824-4d44-8dd0-6dcb0466157b)
5. Potential fix: 
```
                       <h4
                          className="schemaform-array-row-title vads-u-font-size--h5 va-u-outline--none"
                          tabIndex="-1"
                          ref={focusElement}
                        >
                          New {uiOptions.itemName ?? ''}
                        </h4>
```

**Comment by taylorkaren** [comment]
@obliviga: I think there is a word missing in the title of this ticket. Also, can you confirm that this is an issue in the current Forms Library? If so, we should add the Forms Library Support label and epic to this ticket.

**Comment by obliviga** [comment]
@taylorkaren, nice catch -- I have updated the title of the ticket. Confirming that this is an issue in the current Forms Library, as one of the potential fixes is located in `src/platform/forms-system/src/js/review/ArrayField.jsx`.

I'm going to add the Forms Library Support label and epic to this ticket now.

**Comment by obliviga** [comment]
Verified fix on my PR's environment: http://3eed26c38f349ac523596c0c82d844be.review.vetsgov-internal/health-care/apply/application/review-and-submit

![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/4d738fd1-bbd9-4e89-9fea-d10d3f8d8486)
